Assets/Scripts/Actor/EnemyAttackManager.cs

```diff
@@ -25,6 +25,16 @@ public class EnemyAttackManager: MonoBehaviour
         /// The score text box to update and display.
```

Comment:
The changes to this file look misleadingly extensive, but I just inverted an if statement at the beginning of the block to reduce nesting.